src/main/java/ru/yandex/qatools/fsm/impl/YatomataImpl.java

```diff
@@ -15,7 +15,7 @@
 /**
  * @author Ilya Sadykov
  */
-public class YatomataImpl<T> implements Yatomata<T> {
+class YatomataImpl<T> implements Yatomata<T> {
```

Comment:
@smecsia: probably there's better way to get Yatomata instance than mine when using Spring. Is there?